src/UserSpaceInstrumentation/InstrumentProcess.cpp

```diff
@@ -421,10 +426,10 @@ ErrorMessageOr<absl::flat_hash_set<uint64_t>> InstrumentationManager::Instrument
     }
     process_map_.emplace(pid, std::move(process_or_error.value()));
   }
-  OUTCOME_TRY(auto&& instrumented_function_ids,
+  OUTCOME_TRY(auto&& instrumentation_result,
               process_map_[pid]->InstrumentFunctions(capture_options));
 
-  return std::move(instrumented_function_ids);
+  return std::move(instrumentation_result);
```

Comment:
(Doesn't matter but it should be that a return of a local variable already implies a move)

Reply:
Maybe something to do with `instrumentation_result` being a reference of some sort in the `OUTCOME_TRY`, good to know.